blockchain/for-companies/how-to-run-liberland-company.md

```diff
@@ -0,0 +1,132 @@
+## Running Liberland registered companies
+So you have decided to pioneer liberty and decentralization by opening a company and doing business in Liberland.
+
+This article will explain how to create a company or open a subsidiary of another company, 
+how to handle payments either in Liberland itself or internationally or on-chain,
+how to sign and enforce legally binding contracts in Liberland or international jurisdictions, 
+and how to issue stock or other assets and list on the Liberland stock market.
+
+You will need to spend between 10 and 250 LLD for the process.
+
+[Browse existing companies](www.blockchain.liberland.org/home/registries/allCompanies)
+
+[Browse existing Contracts](www.blockchain.liberland.org/home/contracts/overview)
+
+### Choosing personal or company account
+Any person, wallet address of account can create a company in on the Liberland blockchain, however to be officially registered within the nation
+of Liberland it needs to satisfy some minimal requirements.
+
+The first decision is choosing if you want to open a company under your own personal account or create a new company account.
+
+We recommend creating a new company account (wallet address) and doing the process with the company account to keep finances and eventual
```

Comment:
We recommend setting up a new company account (wallet address) to keep business transactions and contracts separate from your personal finances. However, if you're the sole person in your company, you might prefer to use your personal account. This choice depends on your business needs.